### ISSUE TITLE
Suggestion: Change ConvertTo-PSCredential

### ISSUE THREAD

**Issue by colombeen** [dnewsholme/PasswordState-Management#54]
Instead of using
```powershell
Get-PasswordStatePassword -Search test | ConvertTo-PSCredential
```
Turning it into a method on the object which is returned
```powershell
$Result = Get-PasswordStatePassword -Search test
$Result.GetPSCredential()
```

Or maybe even [Type Conversion](https://devblogs.microsoft.com/powershell/understanding-powershells-type-conversion-magic/)
```powershell
[PSCredential] (Get-PasswordStatePassword -Search test)
```

My personal preference would be to use the latter, using type conversion (but i'm not sure if it's even possible to do this easily)

**Comment by colombeen** [comment]
Or maybe even [Type Conversion](https://devblogs.microsoft.com/powershell/understanding-powershells-type-conversion-magic/)

**Comment by dnewsholme** [comment]
A  method would definitely be doable, type conversion will need some investigation. I'll pick this up and have a look/investigation.

**Comment by dnewsholme** [comment]
#57 contains the implementation for this.

**Comment by dnewsholme** [comment]
Closed by #57
